vector/kind.go

```diff
@@ -207,7 +207,23 @@ func ArithOpFromString(op string) int {
 	panic(op)
 }
 
+func OpCodeToString(op int) string {
```

Comment:
Nit: For symmetry with ArithOpFromString.

```suggestion
func ArithOpToString(op int) string {
```